phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/container/control/ContainerVerbsPanel.java

```diff
@@ -33,7 +35,7 @@
     private final ObjectProperty<ApplicationDTO> verbs;
 
     /**
-     * A list of all verbs
+     * A list of all verbs sorted according to their names
```

Comment:
Shouldn't we use `SortedList` instead of `ObservableList`?

Reply:
We can. I just didn't want to use a more concrete type, should I?

Reply:
If we want that the list is sorted, we should also make sure it is in the class contract.

Reply:
Done